README.md

```diff
@@ -1,5 +1,38 @@
 # DataSonnet
 
+## Usage
+
+DataSonnet Mapper is available as a Maven dependency. To include it in your project, use one of the following snippets:
```

Comment:
This should probably mention you can't currently use that unless you do a local build, but it will work soon